src/components/ProfitAndLossTable/ProfitAndLossCompareTable.tsx

```diff
@@ -120,7 +115,12 @@ export const ProfitAndLossCompareTable = ({
           variant={expandable ? 'expandable' : 'default'}
           handleExpand={() => setIsOpen(rowKey)}
```

Comment:
Will this anonymous function cause unnecessary re-rendering? or will setting open/closed will naturally cause the re-rendering so unneeded?